lib/ansible/plugins/connection/ssh.py

```diff
@@ -665,6 +701,49 @@ def _add_args(self, b_command: list[bytes], b_args: t.Iterable[bytes], explanati
         display.vvvvv(u'SSH: %s: (%s)' % (explanation, ')('.join(to_text(a) for a in b_args)), host=self.host)
         b_command += b_args
 
+    def _populate_agent(self):
```

Comment:
```suggestion
    def _populate_agent(self) -> str:
```

*(would be better to return a `pathlib.Path` though)*